src/Servers/Kestrel/Core/src/Internal/Certificates/CertificateConfigLoader.cs

```diff
@@ -114,6 +114,25 @@ private static X509Certificate2 LoadCertificateKey(X509Certificate2 certificate,
         const string SlhDsaShake_256sOid = "2.16.840.1.101.3.4.3.30";
         const string SlhDsaShake_256fOid = "2.16.840.1.101.3.4.3.31";
 
+        const string MLDsa44WithRSA2048PssPreHashSha256Oid = "2.16.840.1.114027.80.9.0";
```

Comment:
These seem wrong? Ref https://datatracker.ietf.org/doc/draft-ietf-lamps-pq-composite-sigs/
> <CompSig> is equal to 2.16.840.1.114027.80.9.1
> id-MLDSA44-RSA2048-PSS-SHA256|<CompSig>.0

That would equal 2.16.840.1.114027.80.9.1.0

Reply:
Wow, good catch. We should probably fix https://github.com/dotnet/runtime/blob/f514fa50f3385d28fad10d3e1ef5b75a8dfa84f5/src/libraries/Common/src/System/Security/Cryptography/Oids.cs then (that's where I copied these from).

cc @bartonjs, @PranavSenthilnathan

Reply:
https://github.com/dotnet/runtime/pull/118795